fs-alert/fs-alert.html

```diff
@@ -1,4 +1,4 @@
-<link rel="import" href="../../polymer/polymer.html">
+<link rel="import" href="../../../polymer/polymer.html">
```

Comment:
The URL issue here is an issue in v8 (the service worker code). It has been fixed and wasn't a problem of this repo.